static/js/components/search/FilterVisibilityToggle.js

```diff
@@ -4,6 +4,11 @@ import _ from 'lodash';
 import { SearchkitComponent } from 'searchkit';
 import Icon from 'react-mdl/lib/Icon';
 
+const FILTER_ID_ADJUST = {
+  "birth_location": "profile.birth_country4",
+  "courses": "program.enrollments.title3"
```

Comment:
i'm kind of worried about this. it seems like the element id's we're getting back from ES/searchkit are almost arbitrary. `profile.birth_country3` changed to `profile.birth_country4` as part of this PR for reasons unknown to me, and i don't know why the id for enrollments is `program.enrollments.title3`. @aliceriot any ideas?